packages/manager/modules/pci/src/projects/project/load-balancer/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "pci_projects_project_load_balancer_list_title": "Load Balancer",
+  "pci_projects_project_load_balancer_list_name": "Nom",
+  "pci_projects_project_load_balancer_list_status": "Statut",
+  "pci_projects_project_load_balancer_list_ipAddres": "Adresse IP",
+  "pci_projects_project_load_balancer_list_healthy": "En bonne santé",
+  "pci_projects_project_load_balancer_list_last_applied_configuration": "Dernière configuration appliquée",
+  "pci_projects_project_load_balancer_list_reqs": "Requis / s",
```

Comment:
What is that supposed to mean ?

Reply:
in mockup it ays Reqs/s. Maybe it means a request

Reply:
it means requests per seconds